python/example_code/dynamodb/GettingStarted/scenario_getting_started_movies.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, dyn_resource):
         :param dyn_resource: A Boto3 DynamoDB resource.
         """
         self.dyn_resource = dyn_resource
+        # The table will be set during the scenario by calling 'exists' with the table name.
```

Comment:
Shouldn't it be?
This is set by 'exists' if the table exists. Otherwise it is set by 'create_table'.

Reply:
Fair point. I updated the description to include both paths.